src/OpenLoco/src/Ui/Screenshot.h

```diff
@@ -3,8 +3,18 @@
 #include <cstdint>
 #include <string>
 
-namespace OpenLoco::Input
+namespace OpenLoco::Ui
 {
+    // TODO: move this
```

Comment:
probably don't need the comment any more?

Reply:
Haha, nice catch! I'll remove it when I rebase for #2235